snowpack/src/commands/dev.ts

```diff
@@ -291,12 +291,27 @@ export async function startServer(
   const inMemoryBuildCache = new Map<string, FileBuilder>();
   let fileToUrlMapping = new OneToManyMap();
 
+  const excludeGlobs = [
```

Comment:
These were the [original exclusions](https://github.com/snowpackjs/snowpack/pull/2876/files#diff-71665e88d7a453bdcd75407adf4c33c4462f08002e3af074bb66487a0311688dL298-L301) prior to the fdir PR.